### PR TITLE
Users route

### PR DESCRIPTION
**Pull request by Maxisthemoose**
User Route stuff

### CODE REVIEW

File: server/api/users.js [hunk 1]
```diff
@@ -0,0 +1,64 @@
+const express = require("express");
+const router = express.Router();
+const User = require("../database/models/User.js");
```
Comment (CreeperPlanet26) [inline server/api/users.js:3]: Should I rename this to "Users" with an "s"?

Comment (Maxisthemoose) [reply server/api/users.js:3]: Yeah sure, if you still can lol

Comment (CreeperPlanet26) [reply server/api/users.js:3]:
```js
const Users = require("../database/models/User.js");
```

Comment (Maxisthemoose) [reply server/api/users.js:3]: Yeah looks good

Comment (CreeperPlanet26) [reply server/api/users.js:3]: alr

**Pull request merged**